cloudbuild.yaml

```diff
@@ -2,18 +2,26 @@ timeout: 1200s
 steps:
   - name: gcr.io/cloud-builders/docker
     args:
+      - HOME=/root
+      - buildx
```

Comment:
tried to make command look like https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/.github/workflows/container-image.yaml#L25